README.md

```diff
@@ -30,6 +30,20 @@ MATTR Wallet SDK React Native / [Exports](modules.md)
 
 # Getting started
 
+## How to get access to MATTR Pi Wallet SDK
+To gain access to the MATTR Pi Wallet SDK, there is a process that must be followed. The following steps are necessary to acquire access to the SDK before it can be used:
```

Comment:
@kimattr Can this be softened please? Something like this:
"To gain access to the MATTR Pi Wallet SDK please follow these steps:"

Reply:
@dylan-paul Please check the revised statement now.